### PR TITLE
used "ReSharper C++" to fix some style and spelling in Application.(h|cpp)

### PR DESCRIPTION
**Pull request by thoys**
## Testing notes

- Perform a smoke test on Interface

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -18,23 +18,12 @@
 #include <glm/gtx/vector_angle.hpp>
 #include <glm/gtc/type_ptr.hpp>
 
-#include <Qt>
-#include <QtCore/QDebug>
-#include <QtCore/QJsonDocument>
-#include <QtCore/QJsonObject>
-#include <QtCore/QObject>
-#include <QtCore/QUrl>
-#include <QtCore/QTimer>
```
Comment (thoys) [inline interface/src/Application.cpp]: Removed a lot of unused includes

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -940,13 +915,13 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer) :
     _applicationStateDevice = std::make_shared<controller::StateController>();
 
     _applicationStateDevice->addInputVariant(QString("InHMD"), controller::StateController::ReadLambda([]() -> float {
-        return (float)qApp->isHMDMode();
+        return static_cast<float>(qApp->isHMDMode());
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: our coding standard says to use C style casts for all "plain old datatypes" -- so I'd really prefer to not have this code styling tool break that convention. Can you disable that option?

Comment (thoys) [inline interface/src/Application.cpp]: c-style casting -> c++

File: interface/src/Application.cpp [hunk 3]
```diff
@@ -1124,7 +1099,7 @@ void Application::showCursor(const QCursor& cursor) {
     _cursorNeedsChanging = true;
 }
 
-void Application::updateHeartbeat() {
+void Application::updateHeartbeat() const {
```
Comment (thoys) [inline interface/src/Application.cpp]: used const wherever possible

File: interface/src/Application.cpp (hunk 2 above)
Comment (thoys) [inline interface/src/Application.cpp]: sure, will do :+1:

**Comment by hifi-gustavo** [comment]
Build console output for commit: b2058376ba38c4df8418197a5013386a6555dad2 available at: https://gist.github.com/34e1c7e379e807fb91fb548ade833675<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7555.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7555.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b2058376ba38c4df8418197a5013386a6555dad2 available at: https://gist.github.com/34e1c7e379e807fb91fb548ade833675<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7555.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7555.dmg

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ Contributing
   git checkout -b new_branch_name 
   ```
 4. Code
-  * Follow the [coding standard](http://docs.highfidelity.io/v1.0/docs/coding-standard)
+  * Follow the [coding standard](https://readme.highfidelity.com/v1.0/docs/coding-standard)
```
Comment (thoys) [inline CONTRIBUTING.md]: Fixed broken link

**Comment by hifi-gustavo** [comment]
Build console output for commit: 142f37da1c55c08fe7c6268aa0b261647f16ca22 available at: https://gist.github.com/34e1c7e379e807fb91fb548ade833675<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7555.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7555.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2be4097bc528ddfde0287b3a9a21ef651ec72b54 available at: https://gist.github.com/34e1c7e379e807fb91fb548ade833675<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7555.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7555.dmg

**Comment by kevinmthomas-carpool** [comment]
PR Build: 7555
Testing Architecture:  64-bit PC Desktop 
Testing OS:  Windows 10
Testing Location:  hifi://music, hifi://sandbox, hifi://playa
Smoke Test:  Successful
Test Results:  Successful smoke test.  Successful import and manipulation of fbx models at both Music and Sandbox.  Consistent graphics results as well at Playa compared to stable release.  Upon moving back to the Music domain, the connection was not successful.  Successful move back to Sandbox and Playa and upon logging out of interface a crash occurred.  Attached are the results.  To create a proper control, tested on build 4614 to which the Music domain did connect properly.  Upon exit of interface a crash also occurred.  Attached are the results.  Results of both the PR and stable Windows build are attached for further inspection.  
[attachments.zip](https://github.com/highfidelity/hifi/files/202902/attachments.zip)

**Pull request merged**